src/korone/database/__init__.py

```diff
@@ -19,33 +19,39 @@ class DatabaseError(Exception):
 
 
 class Database:
-    """Database manager."""
+    """Base class for database."""
 
     path: str
+    """Path to the database file."""
+
     conn: Connection
+    """Connection to the database."""
 
     @classmethod
-    def isopen(cls):
-        """
-        The isopen function is a class method that is used to mark class as having an open connection.
-        It's purpose is to make sure that the connection isn't closed before the class' __del__ method
+    def isopen(cls) -> bool:
+        """The isopen function is a class method that is used to mark class as
+        having an open connection. It's purpose is to make sure that the
+        connection isn't closed before the class' :obj:`__del__` method
```

Comment:
The documentation for this method can be improved, especially because this method is not used by the `__del__` dunder.